images/linux/Ubuntu1804-Readme.md

```diff
@@ -86,30 +87,30 @@
 - nvm 0.39.1
 - OpenSSL 1.1.1  11 Sep 2018
 - Packer 1.8.0
-- Parcel 2.4.0
+- Parcel 2.4.1
 - PhantomJS 2.1.1
 - Podman 3.0.1 (apt source repository: https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable)
-- Pulumi 3.27.0
+- Pulumi 3.28.0
 - R 4.1.3
 - Skopeo 1.2.2 (apt source repository: https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable)
 - Sphinx Open Source Search Server 2.2.11
 - SVN 1.9.7
 - Terraform 1.1.7
 - yamllint 1.26.3
-- yq 4.24.2
+- yq 4.23.1
```

Comment:
We should  investigate. Why was yq downgraded?

Reply:
Looks like there is a bug with GH releases. There is no link to 4.24.2 on the releases page https://github.com/mikefarah/yq/releases even though the direct link works https://github.com/mikefarah/yq/releases/tag/v4.24.2